app/lib/request.rb

```diff
@@ -73,8 +73,6 @@ def perform
       response.body_with_limit if http_client.persistent?
 
       yield response if block_given?
-    rescue => e
-      raise e.class, e.message, e.backtrace[0]
```

Comment:
May we leave this block with changing the code like for `delivery_worker.rb` to keep logs less noisy?

```ruby
    rescue => e
      e.set_backtrace(e.backtrace.first(1))
      raise e
```